flux-dsl/src/main/java/com/influxdb/query/dsl/functions/RangeFlux.java

```diff
@@ -106,6 +106,18 @@ public RangeFlux withStart(@Nonnull final String start) {
         return this;
     }
 
+    /**
+     * @param start Specifies the oldest time (Unix timestamp in seconds) to be included in the results
+     * @return this
+     */
+    @Nonnull
+    public RangeFlux withStart(final long start) {
```

Comment:
Please use `Long` object:

```suggestion
    public RangeFlux withStart(final Long start) {
```